mtools/util/logevent.py

```diff
@@ -205,6 +207,16 @@ def duration(self):
 
         return self._duration
 
+    #SERVER-41349 - get connector from the DNS log line
```

Comment:
connector could be interpreted as something else in this context 🤔

Reply:
same above remove the ticket reference

Reply:
Well, maybe the SERVER ticket reference is ok. It is a public ticket anyway and provides further context about what the intention is for the functionality added.